clientjs/rxhtml.js

```diff
@@ -3064,6 +3064,41 @@ var RxHTML = (function () {
     }
   };
 
+  transforms['dateFmt'] = function(format, d) {
+    if (typeof (d) == "string") {
+      // only transform strings
+      if (d == "1-01-01") return ""
+
+      // if datetime - strip out everything after the T
+      var s = d;
+      var k = s.indexOf('T');
+      if (k >= 0) {
+        s = s.substring(0, k);
+      }
+
+      let [year, month, day] = s.split("-");
+      let p = format.toLowerCase().split("/");
+      const yIndex = p.findIndex(x => x.includes("y"));
+      const mIndex = p.findIndex(x => x.includes("m"));
+      const dIndex = p.findIndex(x => x.includes("d"));
+
+      year = p[yIndex].length == 2 ? year.substring(2,4) : year;
+      month = p[mIndex].length == 1 ? parseInt(month) : month;
```

Comment:
I was going to be snarky about parsing august ("08"), but it seems the latest JS has removed the strange octal behavior: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#converting_mistakes_into_errors